doc/articles/features/using-skia-gtk.md

```diff
@@ -57,9 +57,50 @@ If you want to upgrade **SkiaSharp** to a later version, you'll need to specify
 
 ```xml
 <ItemGroup>
-   <PackagReference Include="SkiaSharp" Version="2.88.0" />
-   <PackagReference Include="SkiaSharp.Harfbuzz" Version="2.88.0" />
-   <PackagReference Include="SkiaSharp.NativeAssets.Linux" Version="2.88.0" />
-   <PackageReference Update="SkiaSharp.NativeAssets.macOS" Version="2.88.0" />
+   <PackagReference Include="SkiaSharp" Version="2.88.3" />
+   <PackagReference Include="SkiaSharp.Harfbuzz" Version="2.88.3" />
+   <PackagReference Include="SkiaSharp.NativeAssets.Linux" Version="2.88.3" />
+   <PackageReference Update="SkiaSharp.NativeAssets.macOS" Version="2.88.3" />
 </ItemGroup>
 ```
+
+### .NET Native AOT support
+
+Building an Uno Platform Skia+GTK app with .NET (7+) Native AOT requires, GtkSharp 3.24.24.38 (or later), or Uno Platform 4.7 (or later).
+
+To build an app with this feature enabled:
+1. Add the following property in your `.csproj`:
+   ```xml
+   <PropertyGroup>
+   		<PublishAot>true</PublishAot>
+	</PropertyGroup>
+   ```
+1. Upgrade your project to net7.0:
+   ```xml
+   <TargetFramework>net7.0</TargetFramework>
+   ```
+1. Add the following items in your `.csproj`:
+   ```xml
+   <ItemGroup>
+      <TrimmerRootAssembly Include="MyApp.Skia.Gtk" />
+		<RdXmlFile Include="MyApp.rd.xml" />
+		<RuntimeHostConfigurationOption Include="Switch.System.Reflection.Assembly.SimulatedCallingAssembly" Value="true" />
+   </ItemGroup>
+   ```
+1. Create a file name `MyApp.rd.xml`, and place it next to your csproj:
+   ```xml
+   <Directives>
+      <Application>        
+         <Assembly Name="GdkSharp" />
+         <Assembly Name="GtkSharp" />
+      </Application>
+   </Directives>
```

Comment:
RD.XML is unsupported and can break any time. What this is doing can be expressed using 100% supported route by adding this to the project file:

```xml
<ItemGroup>
  <TrimmerRootAssembly Include="GtkSharp: />
  <TrimmerRootAssembly Include="GdkSharp: />
</ItemGroup>
```

Reply:
Thanks for the update! (again, it's very much appreciated!)